src/duktape/duk_config.h

```diff
@@ -2338,7 +2338,7 @@ typedef FILE duk_file;
 #endif
 #endif
 
-/* Compiler specific hackery needed to force struct size to match aligment,
+/* Compiler specific hackery needed to force struct size to match alignment,
```

Comment:
I don't think it makes sense to correct vendored library https://duktape.org/

Reply:
I'll merge it anyway. Worst case they will reappear when we update used duktape version.